test/framework/fixtures/restClientFixture.go

```diff
@@ -271,6 +271,34 @@ func (f *RestClientFixture) WaitForAllTxnsToConfirm(roundTimeout uint64, txidsAn
 	return true
 }
 
+// WaitForAccountFunded waits until either the passed account gets non-empty balance
+// or until the passed roundTimeout passes
+// or until waiting for a round to pass times out
+func (f *RestClientFixture) WaitForAccountFunded(roundTimeout uint64, accountAddress string) (err error) {
```

Comment:
I think it is better to move this inside the test file.
The utility of this function in another situation is very limited, because it is not necessary to have the `accountAddress` in the `f.AlgodClient` wallet in other situations.
 
Alternatively, there is a simpler (perhaps less efficient) but more general function: to check for the pending transactions and wait until all pending transactions are flushed. 

Also, a bit more intrusive check (but more efficient) would be to get the txid from `SendPaymentFromUnencryptedWallet`, and pass it here, and wait for that transaction to be accepted.

Reply:
I though about returning list of txnids from `setupActors` but it looked messy. Ended up with this approach.